### ISSUE TITLE
Migrate to wiki when refined

### ISSUE THREAD

**Issue by NickSeagull** [DataHaskell/numeric-libs-overview#1]
This issue is for serving as a reminder to migrate all the conclusions we draw from here, in a refined format to the wiki in the future.

**Comment by mdibaiee** [comment]
@NickSeagull now we have the content on [docs](http://datahaskell.org/docs) and this repository is going to hold the benchmarks only.

**Comment by NickSeagull** [comment]
I was thinking about putting the benchmarks in some dropdowns, what do you thing @mdibaiee ?

**Comment by mdibaiee** [comment]
@NickSeagull Um, you mean dropdowns in docs and moving the HTML there? What about the source codes?
I think having benchmarks in a separate repository helps people evaluate them much more easily, they should not have to clone the docs repository to run benchmarks and see the results 🤔

**Comment by NickSeagull** [comment]
That is true, @mdibaiee, I think we should just clean the repo up a bit 😄

**Comment by mdibaiee** [comment]
@NickSeagull I will rewrite a simple README for the repo to point out the docs and a small guide on how to write benchmarks. :grin:
